serenata_toolbox/federal_senate/federal_senate_dataset.py

```diff
@@ -0,0 +1,101 @@
+import os.path
+from urllib.request import urlretrieve
+import numpy as np
+import pandas as pd
+
+from datetime import date
+
+class FederalSenateDataset:
+    URL = 'http://www.senado.gov.br/transparencia/LAI/verba/{}.csv'
+    FIRST_YEAR = 2008
+    NEXT_YEAR = date.today().year + 1
+
+    def __init__(self, path):
+        self.path = path
+
+    def fetch(self):
+        for year in range(self.FIRST_YEAR, self.NEXT_YEAR):
+            url = self.URL.format(year)
+            file_path = os.path.join(self.path, 'federal-senate-{}.csv'.format(year))
+            urlretrieve(url, file_path)
+
+    def translate(self):
+        filenames = ['federal-senate-{}.csv'.format(year) for year in range(self.FIRST_YEAR, self.NEXT_YEAR)]
```

Comment:
I'd add something as `YEAR_RANGE` as a class constant to avoid repeating this `range(…)`.